environment-templates/ecs-ec2-env/v1/infrastructure/outputs.tf

```diff
@@ -0,0 +1,35 @@
+output "vpc_id" {
```

Comment:
The convention we landed on for output variables is Capitalized with a descriptor suffix where appropriate e.g. `PublicSubnetOneId`, `VpcId` etc